lighty-core/lighty-controller/src/main/java/io/lighty/core/controller/impl/LightyControllerImpl.java

```diff
@@ -219,15 +241,25 @@ private static final void initSunXMLWriterProperty() {
     @Override
     protected boolean initProcedure() {
         final long startTime = System.nanoTime();
+        final boolean k8sDeployment = isKubernetesDeployment(this.actorSystemConfig);
 
         //INIT actor system provider
         this.actorSystemProvider = new ActorSystemProviderImpl(this.actorSystemClassLoader,
-                QuarantinedMonitorActor.props(() -> {}), this.actorSystemConfig);
+                QuarantinedMonitorActor.props(() -> {
+                }), this.actorSystemConfig);
+
+        this.akkaManagement = AkkaManagement.get(actorSystemProvider.getActorSystem());
+        akkaManagement.start();
+        //INIT cluster bootstrap, if this is Kubernetes Deployment
+        if (k8sDeployment) {
```

Comment:
boolean if ... given OOP, is this the case where we want to have a subclass callout and a dedicated subclass?